logger-mongo-hook.go

```diff
@@ -25,31 +25,29 @@ import (
 	"gopkg.in/mgo.v2/bson"
 )
 
+type mongoLogger struct {
+	Enable    bool     `json:"enable"`
+	Addr      string   `json:"address"`
+	LogLevels []string `json:"logLevels"`
+}
+
 // mongoDB collection
 type mongoDB struct {
```

Comment:
All these code has to be rewritten completely. We should never have structs like mongoDB and member named c